frontend/app/src/components/accounts/management/types/AddressAccountForm.vue

```diff
@@ -84,15 +84,22 @@ const save = async () => {
       }
     }
   } catch (e: any) {
-    set(errorMessages, deserializeApiErrorMessage(e.message) || {});
-
-    await setMessage({
-      description: tc('account_form.error.description', 0, {
-        error: e.message
-      }).toString(),
-      title: tc('account_form.error.title'),
-      success: false
-    });
+    if (e instanceof ApiValidationError) {
+      const errors = e.getValidationErrors({});
+
+      if (typeof errors === 'string') {
+        setMessage({
+          description: tc('account_form.error.description', 0, {
+            error: errors
+          }).toString(),
+          title: tc('account_form.error.title'),
+          success: false
+        });
+      } else {
+        set(errorMessages, errors);
+      }
+    }
+
```

Comment:
Okay it was my bad.
when I saw `getValidationErrors` returns `ValidationErrors | string`, I thought it will return the string if there's no error object there.
And also, isn't all 400 errors will be thrown as `ApiValidationError`?

Reply:
Okay I see we can also get 409 which is string-only message. I will adjust all of them.